src/view/configuration/configuration.jsx

```diff
@@ -55,7 +55,8 @@ const getInstanceDefaults = initInfo => ({
   cookieDestinationsEnabled: true,
   prehidingStyle: "",
   contextGranularity: contextGranularityEnum.ALL,
-  context: contextOptions
+  context: contextOptions,
+  migrateIds: false
 });
```

Comment:
We talked a while back about merging the alloy and alloy-extension code bases.  Looking at this instanceDefaults object, there is an opportunity here to run an automated test to make sure these stay in sync with the validators in alloy.